Packs/CortexXDR/.pack-ignore

```diff
@@ -26,4 +26,7 @@ ignore=PB110
 ignore=PB110
 
 [file:PaloAltoNetworks_Cortex_XDR_Incident_Sync.yml]
-ignore=PB110
\ No newline at end of file
+ignore=PB110
+
+[file:XDR_iocs_-_Test.yml]
```

Comment:
why are we keeping it? you can delete the old test

Reply:
Because if we will have a way to run `fetch-indicator` (e.g. like fetch-incidents).
this test will be really useful.

Reply:
in that case:
* open a dev-task for it
* remove it from the ignore
* add it to the skipped tests and link to the issue